packages/channel-provider/package.json

```diff
@@ -14,7 +14,6 @@
     "prepublish": "webpack",
     "build": "webpack",
     "build:typescript": "tsc -b .",
-    "prepare": "yarn build",
```

Comment:
`channel-client` and `embedded-wallet` import `channel-provider`. `eslint` will fail during the circle `build` job.

Reply:
The prepare command should probably be switched to run `yarn build:typescript`